src/main/kotlin/xyz/crossward/service/UserService.kt

```diff
@@ -86,7 +83,6 @@ class UserService(
         return savedUser
     }
 
-    fun userExists(userId: String): Boolean {
-        return userRepository.existsById(userId)
-    }
+    fun userExists(userId: String): Boolean =
+        userRepository.existsById(userId)
```

Comment:
Just curious - why'd you remove the return

Reply:
I'm using the expression body syntax which allows you to use an `=` sign instead of a return and a pair of brackets. You can only do them when the function body is one line.